CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+1.9.0
+
+* Feature: BREAKING CHANGE TO LANGUAGE AND API: Add `constructors` keyword
```

Comment:
This is not documented in the Wiki yet (since I haven't yet migrated the Haskell tutorial over to the Wiki) but is documented in the Haskell tutorial.  I will add something along the lines of "See the updated Haskell tutorial for more details" if that's alright